packages/typespec-test/test/eventgrid_modular/spec/main.tsp

```diff
@@ -17,8 +16,7 @@ enum ServiceApiVersions {
 )
 
 @service({
-  title: "AzureMessagingEventGridClient",
-  version: "2023-06-01-preview"
```

Comment:
does this mean we can not use service decorator to specify api version ?

Reply:
you could see here: https://microsoft.github.io/typespec/release-notes/release-2023-05-10#typespecversioning